non-oss/pharos_pro/addons/kontena-lens/addon.rb

```diff
@@ -36,9 +37,11 @@
     apply_resources(
       host: host,
       email: config.tls&.email,
+      tls_enabled: tls_enabled?,
       user_management: user_management_enabled?
     )
-    message = "Kontena Lens is configured to respond at: " + pastel.cyan("https://#{host}")
+    protocol = !tls_enabled? 'http' : 'https'
```

Comment:
Should we flip this?

```suggestion
    protocol = tls_enabled? 'https' : 'http'
```

Reply:
yes...better :+1:

Reply:
Actually, does this need double questionmarks? I resolved this in GH UI.

```suggestion
    protocol = tls_enabled? ? 'https' : 'http'
```

Trying in terminal now...

Reply:
Yeah, it needs the latter suggestion.